a/afero/build_info.json

```diff
@@ -2,6 +2,8 @@
     "package_name" : "afero",
     "github_url":"https://github.com/spf13/afero",
     "version": "v1.9.3",
+    "default_branch": "master",
```

Comment:
Remind me - the reason why we need to set this in the build_info files rather than fetch it programmatically is because of the concern around the number of GitHub API calls, correct?

Reply:
Correct.